src/SessionInstance.php

```diff
@@ -69,7 +75,12 @@ protected function init()
 
         session_name($this->name);
 
-        session_start();
+        if ($this->sessionID) {
+            session_id($this->sessionID);
+        }
+
+        # Start the session up, but ignore the error about headers already being sent
+        @session_start();
```

Comment:
In my scenario (WebSocket), SessionInstance object need to be made multiple times. So this line will cause headers already sent error. Hence I did this.

I think it's better to warn users to make SessionInstance objects before output. This'll help this library to be used in non-HTTP projects too.